inv-management/src/Item.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+public class Item {
+    private String name;
+    private Integer price;
+    private Integer stock;
+    private Scanner input = new Scanner(System.in);
+
+    public Item(){
+        System.out.format("name:");
+        this.name = input.nextLine();
```

Comment:
While this may seem like a good place to put the input it will cause issues later.  At this point you have no ability save this class as you cannot create the class without asking for input from the user.  So say like you were saving the data to a file and then reading it back from the file you have no path to create `Item` from existing data.  This is better put in a method that is called after an item instance has been created.